src/Session.hs

```diff
@@ -69,7 +70,9 @@ kill ghci = ignored $ do
     debugShutdown "Before terminateProcess"
     ignored $ terminateProcess $ process ghci
     debugShutdown "After terminateProcess"
-
+    -- Ctrl-C after a tests keeps the cursor hidden,
+    -- `setSGR []`didn't seem to be enough
```

Comment:
Can you add a link to this ticket, since I expect I'll want to know more about this one day.